src/Voyager.php

```diff
@@ -160,10 +160,27 @@ public function formFields()
 
     public function setting($key, $default = null)
     {
+        $globalCache = config('voyager.settings.cache', false);
+
+        if ($globalCache && Cache::tags('settings')->has($key)) {
+            return Cache::tags('settings')->get($key);
+        }
+
         if ($this->setting_cache === null) {
```

Comment:
Was it intentional to leave the per-request/local cache in place?  I feel like this would be much simpler if it was simply replaced with `Cache::` calls like you've already done.

Reply:
It's been a while, but if I recall correctly, it was to disallow users to use the caching function, if they for some reason wanted to do that. It can be stripped out without issues